rpc/javascript.go

```diff
@@ -399,6 +399,11 @@ web3._extend({
 			call: 'debug_writeMemProfile',
 			params: 1
 		}),
+		new web3._extend.Method({
+			name: 'replayTransaction',
+			call: 'debug_replayTransaction',
+			params: 4
+		})
```

Comment:
Hmm, you are indeed right. Odd that this part didn't sort the method names. Anyway, then that should definitely be done, but in another PR. This should be fine as is this way.